packages/core/docs/general/support.md

```diff
@@ -0,0 +1,32 @@
+# Support
+
+## How to get help?
+
+First of all, feel invited to join our community at [Discord](http://discord.vuestorefront.io/) - this is the place, where you can ask your questions, start interesting discussions, and meet Vue Storefront Core Team Members.
```

Comment:
```suggestion
Feel invited to join our community at [Discord](http://discord.vuestorefront.io/) - this is the place, where you can ask your questions, start interesting discussions, and meet Vue Storefront Core Team Members.
```
Think it's kinda redundant, especially when we start with "First" in next section. Just an idea, tho :)